resources/public/api/conf/1.0/application.yaml

```diff
@@ -1564,21 +1606,27 @@ paths:
 
         \nAs data is scoped, the JSON payload that you send should only contain the data items specific to the use case.
 
-        \nAs filtering logic does not form part of this API, it will return the exact JSON that was submitted."
+        \nAs filtering logic does not form part of this API, it will return the exact JSON that was submitted.
+        
+        \n#### Authorisation
+        
+        \nThis endpoint is [application-restricted](https://developer.service.hmrc.gov.uk/api-documentation/docs/authorisation/application-restricted-endpoints) and requires `Authorization` header containing an OAUTH 2.0 Bearer Token.
+        
+        \nSee also [fraud prevention](https://developer.service.hmrc.gov.uk/guides/fraud-prevention/)."

```

Comment:
ℹ️ You can do a multiline string by doing:

```yaml
  description: >-
    Use this endpoint to create...
```

Then you won't have to spam `\n` all the time.

Reply:
some of these formatting are not possible in the case here
<img width="777" alt="Screenshot 2023-03-02 at 14 49 25" src="https://user-images.githubusercontent.com/58558440/222462503-bab81477-20d1-456f-b25d-2cebbf854e5e.png">